src/NodaTime.Test/LocalDateTest.cs

```diff
@@ -57,6 +58,16 @@ public void XmlSerialization_Invalid(string xml, Type expectedExceptionType)
         }
 
         [Test]
+        public void Construction_NullCalendar_Throws()
+        {
+            Assert.Multiple(() =>
```

Comment:
Interesting - I haven't seen Assert.Multiple before.

Reply:
Nice, isn't it? :) It's a new-ish feature in NUnit; I guess they borrowed that from xunit.